sdk/storage/azure-storage-files-shares/test/ut/share_file_client_test.cpp

```diff
@@ -1179,23 +1179,30 @@ namespace Azure { namespace Storage { namespace Test {
     auto sourceBlobClient = containerClient.GetBlockBlobClient(blobName);
     sourceBlobClient.Upload(memBodyStream);
 
+    std::shared_ptr<Azure::Core::Credentials::TokenCredential> oauthCredential
+        = std::make_shared<Azure::Identity::ClientSecretCredential>(
+            AadTenantId(),
+            AadClientId(),
+            AadClientSecret(),
+            InitStorageClientOptions<Azure::Identity::ClientSecretCredentialOptions>());
+    auto clientOptions = InitStorageClientOptions<Files::Shares::ShareClientOptions>();
+    clientOptions.ShareTokenIntent = Files::Shares::Models::ShareTokenIntent::Backup;
+
     auto destFileClient
```

Comment:
We remove a test case where copy destination was authenticated with shared key and added a new test case.

Can we do this without deleting the old test case?